### ISSUE TITLE
Access for downsteam-task data meta file

### ISSUE THREAD

**Issue by XiejiLi** [suinleelab/MONET#9]
Hi, authors,
  Thank you for you work for explainable AI in dermoatology filed. Currently I am reproducing your result in multiple downstream tasks. I tried to dive into a [notebook](https://github.com/suinleelab/MONET/blob/main/experiments/concept_annotation.ipynb), but it's hard to follow your work without a downstream dataset based on F17k and DDI, would you offer the meta file or code to process these downsteam-task dataset?
   Thank a lot for presenting a inspired work for us.

**Comment by XiejiLi** [comment]
Also, I reproduce the result of F17k disease label annotation. When i processed f17k dataset,the positive example is less than 30 being exlcuded, these label left:

melanoma
nevus
basal cell carcinoma
actinic keratosis
benign keratosis
dermatofibroma
vascular lesion
malignant
seborrheic keratosis
squamous cell carcinoma
lentigo NOS
lichenoid keratosis

there are still 2 labels not found(currently 12 labels in 46051 images), could you provide the label columns you use in this experiment?

**Comment by chanwkimlab** [comment]
Hi, thank you for your interest in our work! The Fitzpatrick17k dataset contains three columns related to skin conditions: `label`, `nine_partition_label`, and `three_partition_label`. We used the `label` column, as it provides the most fine-grained disease classification. Also, you can find scripts for preprocessing datasets [here]( https://github.com/suinleelab/MONET/tree/main/scripts/preprocess). I hope this helps!
